<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalogChannelDialog</class>
 <widget class="QDialog" name="AnalogChannelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Analog Channel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="channelName">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="rxFrequencyLabel">
         <property name="text">
          <string>Rx Frequency</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="rxFrequency"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="txFrequencyLabel">
         <property name="text">
          <string>Tx Frequency</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txFrequency"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="powerLabel">
         <property name="text">
          <string>Power</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="power">
         <property name="currentText">
          <string>High</string>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Max</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>High</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mid</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Low</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Min</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="txTimeoutLabel">
         <property name="text">
          <string>Tx Timeout</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="txTimeout">
         <property name="maximum">
          <number>555</number>
         </property>
         <property name="singleStep">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="rxOnlyLabel">
         <property name="text">
          <string>Rx Only</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="rxOnly"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="scanListLabel">
         <property name="text">
          <string>Scan List</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="scanList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="txAdmitLabel">
         <property name="text">
          <string>Tx Admit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="txAdmit">
         <item>
          <property name="text">
           <string>Always</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Channel Free</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tone</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="squelchLabel">
         <property name="text">
          <string>Squelch</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="squelch">
         <property name="maximum">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="rxToneLabel">
         <property name="text">
          <string>Rx Tone</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="rxTone"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="txToneLabel">
         <property name="text">
          <string>Tx Tone</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="txTone"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="bandwidthLabel">
         <property name="text">
          <string>Bandwidth</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="bandwidth">
         <item>
          <property name="text">
           <string>Narrow (12.5 kHz)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wide (25 kHz)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AnalogChannelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AnalogChannelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
